.travis.yml

```diff
@@ -32,6 +32,8 @@ matrix:
       env: FEATURE=clippy-sandbox
     - rust: nightly-2018-01-22
       env: FEATURE=clippy-testkit
+    - rust: nightly-2018-01-22
+      env: FEATURE=clippy-exonum-time
```

Comment:
Something that bugged me for a while:

- I think it could be better to define jobs in the Travis config (see #459, where I've done this). IMO, it's more readable than having `switch`es in `install` and `script` hooks.
- Does it really make sense to have multiple `clippy` and test jobs? I agree that clippy jobs should be separated from test ones, because they seemingly overwrite each other's artifacts; but I see little sense in splitting clippy / test jobs among services.

@DarkEld3r, could you comment on the topic?

Reply:
I had in mind the "optimization" of Travis build script, but thought of it as a separate task/pull request. 

I also think that jobs are easier to read and maintain, so we can switch to them already if @SerhiiOryshych doesn't mind.

Not sure about merging `clippy` jobs into one: it can take a long time, but it worth to try.